src/Components/Users/ManageUsers.tsx

```diff
@@ -282,16 +281,15 @@ export default function ManageUsers() {
                   {user.user_type === "Doctor" && (
                     <>
                       <div className="col-span-1">
-                        <UserDetails
-                          id="doctor-qualification"
-                          title="Qualification"
-                        >
-                          {user.doctor_qualification ? (
+                        <UserDetails id="qualification" title="Qualification">
+                          {user.qualification ? (
                             <span className="font-semibold">
-                              {user.doctor_qualification}
+                              {user.qualification}
```

Comment:
i18n

Reply:
how can I do for this ,I can understand for static data like title placeholder label ?

Reply:
ignore